<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mayo::WidgetGrid</class>
 <widget class="QWidget" name="Mayo::WidgetGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="check_ShowGrid">
       <property name="text">
        <string>Show Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_Plane">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Plane: XOY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plane: ZOX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plane: YOZ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plane: Custom</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_Main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="btn_Config">
            <property name="text">
             <string>Configuration</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::RightArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_Config" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_Type">
             <property name="text">
              <string>Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="combo_Type">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Rectangular</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Circular</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QStackedWidget" name="stack_Config">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="page_Rectangular">
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
               <item row="0" column="2">
                <widget class="QDoubleSpinBox" name="edit_RectOriginY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Step</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="edit_RectStepX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>X</string>
                 </property>
                 <property name="minimum">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QDoubleSpinBox" name="edit_RectStepY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>Y</string>
                 </property>
                 <property name="minimum">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Size</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="edit_RectSizeX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QDoubleSpinBox" name="edit_RectSizeY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Rotation</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="edit_RectRotation">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="suffix">
                  <string>°</string>
                 </property>
                 <property name="minimum">
                  <double>-180.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>5.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QWidget" name="widget_2" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Offset</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="edit_RectOffset">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="suffix">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="edit_RectOriginX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_RectOrigin">
                 <property name="text">
                  <string>Origin</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_Circular">
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Origin</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="edit_CircOriginX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QDoubleSpinBox" name="edit_CircOriginY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="prefix">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Radius</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="edit_CircRadius">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QWidget" name="widget" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Radius Step</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="edit_CircRadiusStep">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Division</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="edit_CircDivision">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Rotation</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="edit_CircRotation">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="suffix">
                  <string>°</string>
                 </property>
                 <property name="minimum">
                  <double>-180.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>5.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Offset</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="edit_CircOffset">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="suffix">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="btn_Graphics">
            <property name="text">
             <string>Graphics</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::RightArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_Graphics" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="1" column="1">
            <widget class="QToolButton" name="btn_Color">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_ColorTenth">
             <property name="text">
              <string>Tenth Color</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="btn_ColorTenth">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Mode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QComboBox" name="combo_DrawMode">
             <item>
              <property name="text">
               <string>Lines</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Points</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="check_ColorTenth">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>check_ShowGrid</tabstop>
  <tabstop>combo_Plane</tabstop>
  <tabstop>btn_Config</tabstop>
  <tabstop>combo_Type</tabstop>
  <tabstop>edit_RectOriginX</tabstop>
  <tabstop>edit_RectOriginY</tabstop>
  <tabstop>edit_RectStepX</tabstop>
  <tabstop>edit_RectStepY</tabstop>
  <tabstop>edit_RectSizeX</tabstop>
  <tabstop>edit_RectSizeY</tabstop>
  <tabstop>edit_RectRotation</tabstop>
  <tabstop>edit_RectOffset</tabstop>
  <tabstop>btn_Graphics</tabstop>
  <tabstop>combo_DrawMode</tabstop>
  <tabstop>btn_Color</tabstop>
  <tabstop>btn_ColorTenth</tabstop>
  <tabstop>edit_CircOriginX</tabstop>
  <tabstop>edit_CircOriginY</tabstop>
  <tabstop>edit_CircRadius</tabstop>
  <tabstop>edit_CircRadiusStep</tabstop>
  <tabstop>edit_CircDivision</tabstop>
  <tabstop>edit_CircRotation</tabstop>
  <tabstop>edit_CircOffset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
